src/lib/utils/find.ts

```diff
@@ -6,14 +6,18 @@
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find
 function find<TItem>(
   items: TItem[],
-  predicate: (value: TItem, index: number, obj: TItem[]) => boolean,
-  thisArg?: any
+  predicate: (value: TItem, index: number, obj: TItem[]) => boolean
 ): TItem | undefined {
-  if (!Array.prototype.find) {
-    return items.filter(predicate, thisArg)[0];
+  let value: TItem;
+  for (let i = 0; i < items.length; i++) {
+    value = items[i];
+    // inlined for performance: if (Call(predicate, thisArg, [value, i, list])) {
+    if (predicate(value, i, items)) {
+      return value;
+    }
   }
 
-  return items.find(predicate, thisArg);
+  return undefined;
```

Comment:
I could write a more complicated error check in the babel config so that it could for example allow Array.find exactly twice, but that seemed much more complicated than rewriting this to an as-efficient version as Array.find

Reply:
Makes sense